digital-deck-server/models/dbConnection.js

```diff
@@ -0,0 +1,28 @@
+require('dotenv').config({path:'./../digital-deck-server/ENV.env'});
```

Comment:
Does this require anything extra locally? Or should it work automatically?

Reply:
requires a .env file. Which I can share with you. This is for security purposes.